.github/ISSUE_TEMPLATE/helm_v3_bug_report.md

```diff
@@ -0,0 +1,33 @@
+---
+name: Helm v3 bug report
+about: Create a bug report for the (alpha) Flux Helm operator with Helm v3 support
```

Comment:
I would remove alpha from here, this template is here to stay for a long time :)